.github/policies/resourceManagement.yml

```diff
@@ -277,34 +277,6 @@ configuration:
       then:
       - closeIssue
 
-    - description: Add "untriaged" label on new issues
-      triggerOnOwnActions: false
-      if:
-      - payloadType: Issues
-      - not:
-          hasLabel:
-            label: untriaged
-      - and:
-          - isAction:
-              action: Opened
-      then:
-      - addLabel:
-          label: untriaged
-
-    - description: Remove "untriaged" label from issues when closed or added to a milestone

```

Comment:
Should we clear out all untriaged labels on issues in that case. I've got a script we can run to match the status field with this tag that we might want to run before. Noticed a few issues in the roslyn repo where this somehow doesn't match.

Reply:
I would rather not do that. Such issue manipulation triggers notifications for all subscribed people which I don't think is worth it in this case.